### PR TITLE
SSR-2824 User guide for ReadOnlyTable

### PR DESCRIPTION
**Pull request by nmanhit**
## Checklist

- [ ] Read CONTRIBUTING.md([for v1](https://github.com/kintone-labs/kintone-ui-component/master/CONTRIBUTING.md), [for v0](https://github.com/kintone-labs/kintone-ui-component/v0_dev/CONTRIBUTING.md))
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed lint and test command.

### CODE REVIEW

File: docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md [hunk 1]
```diff
@@ -7,4 +7,104 @@ original_id: readonly-table
...
+      index: "3",
+      name: "New York",
+      country: "USA",
+      population: "8,400,000",
+      coordinates: "40.712776, -74.005974",
+    },
+  ],
+});
+space.appendChild(readOnlyTable);
+```
```
Comment (voduchau) [inline docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md:110]: add empty line end of this file

File: docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md [hunk 2]
```diff
@@ -7,4 +7,104 @@ original_id: readonly-table
...
+const space = kintone.app.record.getSpaceElement('space');
+
+const readOnlyTable = new Kuc.ReadOnlyTable({
+  label: "ReadOnlyTable",
+  rowsPerPage: 3,
+  pagination: true,
+  columns: [
+    {
+      title: "Number",
+      field: "index",
```
Comment (voduchau) [inline docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md:66]: In sample code, please use single qoutes

File: docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md (hunk 1 above)
Comment (voduchau) [reply docs/document/website/versioned_docs/version-1.8.0/components/desktop/readonly-table.md:110]: please check and update at docs/components/desktop/readonly-table.js too

File: docs/document/docs/components/desktop/readonly-table.md [hunk 3]
```diff
@@ -24,11 +28,13 @@ Here is a list of properties that can be used for modifying the component:
...
 | visible | boolean | true | Show/Hide the component | |
-| columns | Array\<Columns\> | []  | Array of data to be displayed in the table header | Will result an error if the value for columns is not an array |
-| Columns.header | Object | Null | Table header object | |
-| Columns.header.text | String | Null | Text to be displayed in the table header | |
-| Columns.visible | Boolean | Null | Show/Hide the column | |
-| data | Array\<Array\<string\>\> | []  | Two-dimensional array of data to be displayed on table body | Will result an error if the data is not an array |
+| columns | Array\<Column\> | []  | Array of data to be displayed in the table header | Will result an error if the value for columns is not an array |
+| Column.title | String | "" | The header name of each column | |
+| Column.field | String | "" | The key of the data object | |
+| Column.visible | Boolean | true | Show/Hide the column | |
```
Comment (voduchau) [inline docs/document/docs/components/desktop/readonly-table.md:34]: Separate column property to a table

File: docs/document/docs/components/desktop/readonly-table.md [hunk 4]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| className | string | ""  | Component class name. | |
+| id | string | ""  | Component id name. | |
+| label | string | ""  | Text description for the component. | Label will not be displayed if unspecified or left empty. |
+| rowsPerPage | number | 5 | Number of table rows per page. | <ul><li>Round off to the nearest whole number. </li><ul> <li> `{rowsPerPage: 10}` Display up to 10 rows per page. </li><li> `{rowsPerPage: 20.5}` Display up to 21 rows per page. </li> <li> `{rowsPerPage: 20.4}` Display up to 20 rows per page. </li></ul><li>Throw an error if `rowsPerPage` is not a positive number in constructor and setter.</li><ul>|
+| pagination | boolean | true | Show/Hide the pagination. | <ul><li>`{pagination: false}` Pagination is hidden, all rows are displayed.</li><li>`{pagination: true}` Only the number of rows set in rowsPerPage are displayed.</li></ul> |
+| visible | boolean | true | Show/Hide the component. | |
+| columns | Array\<[Columns](#column)\> | []  | Column data of the component. | Throw an error if the `columns` is not an array. |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:33]:
```suggestion
| columns | Array\<[Column](#column)\> | []  | Column data of the component. | Will result an error if the value of `columns` is not an array. |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 5]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| data | Array\<object\> | []  | Row data of the component. | Throw an error if the `data` is not an array. |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:34]:
```suggestion
| data | Array\<object\> | []  | Row data of the component | Will result an error if the value of `data` is not an array. |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 6]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
-| className | string | ""  | Component class name | |
-| id | string | ""  | Component id name | |
-| label | string | ""  | Label for the component | Label will not be displayed if unspecified or left empty |
-| visible | boolean | true | Show/Hide the component | |
-| columns | Array\<Columns\> | []  | Array of data to be displayed in the table header | Will result an error if the value for columns is not an array |
-| Columns.header | Object | Null | Table header object | |
-| Columns.header.text | String | Null | Text to be displayed in the table header | |
-| Columns.visible | Boolean | Null | Show/Hide the column | |
-| data | Array\<Array\<string\>\> | []  | Two-dimensional array of data to be displayed on table body | Will result an error if the data is not an array |
+| className | string | ""  | Component class name. | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:27]:
```suggestion
| className | string | ""  | Component class name | |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 7]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| id | string | ""  | Component id name. | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:28]:
```suggestion
| id | string | ""  | Component id name | |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 8]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| rowsPerPage | number | 5 | Number of table rows per page. | <ul><li>Round off to the nearest whole number. </li><ul> <li> `{rowsPerPage: 10}` Display up to 10 rows per page. </li><li> `{rowsPerPage: 20.5}` Display up to 21 rows per page. </li> <li> `{rowsPerPage: 20.4}` Display up to 20 rows per page. </li></ul><li>Throw an error if `rowsPerPage` is not a positive number in constructor and setter.</li><ul>|
+| pagination | boolean | true | Show/Hide the pagination. | <ul><li>`{pagination: false}` Pagination is hidden, all rows are displayed.</li><li>`{pagination: true}` Only the number of rows set in rowsPerPage are displayed.</li></ul> |
+| visible | boolean | true | Show/Hide the component. | |
+| columns | Array\<[Columns](#column)\> | []  | Column data of the component. | Throw an error if the `columns` is not an array. |
+| data | Array\<object\> | []  | Row data of the component. | Throw an error if the `data` is not an array. |
+
+#### Column
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| field | string | ""  | Key of the data object. | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:39]:
```suggestion
| field | string | ""  | Key of the data object | |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 9]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| title | string | ""  | Header name of the column. | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:40]:
```suggestion
| title | string | ""  | Header name of the column | |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 10]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| visible | boolean |  true  | Show/Hide the column. | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:41]:
```suggestion
| visible | boolean |  true  | Show/Hide the column | |
```

File: docs/document/docs/components/desktop/readonly-table.md [hunk 11]
```diff
@@ -46,37 +56,58 @@ Here is a list of available constructors:
...
+      population: '8,400,000',
+      coordinates: '40.712776, -74.005974',
+    },
   ],
-  className: 'options-class',
-  id: 'options-id',
-  visible: true
 });
 space.appendChild(readOnlyTable);
 ```
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:113]: Please update with below including every props.

```
const Kuc = Kucs['1.x.x'];

const space = kintone.app.record.getSpaceElement('space');

const readOnlyTable = new ReadOnlyTable({
  label: 'ReadOnlyTable',
  columns: [
    {
      title: 'Number',
      field: 'index',
    },
    {
      title: 'City',
      field: 'name',
    },
    {
      title: 'Country',
      field: 'country',
    },
    {
      title: 'Population',
      field: 'population',
    },
    {
      title: 'Coordinates',
      field: 'coordinates',
    }
  ],
  data: [
    {
      index: '1',
      name: 'HoChiMinh',
      country: 'Vietnam',
      population: '8,371,000',
      coordinates: '10.762622, 106.660172',
    },
    {
      index: '2',
      name: 'Tokyo',
      country: 'Japan',
      population: '14,000,000',
      coordinates: '35.689487, 139.691711',
    },
    {
      index: '3',
      name: 'New York',
      country: 'USA',
      population: '8,400,000',
      coordinates: '40.712776, -74.005974',
    }
  ],
  className: 'sample-class',
  id: 'sample-id',
  visible: true,
  pagination: true,
  rowsPerPage: 3,
});
space.appendChild(readOnlyTable);
```

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please update following review comments.

File: docs/document/docs/components/desktop/readonly-table.md [hunk 12]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| data | Array\<object\> | []  | Row data of the component. | Throw an error if the `data` is not an array. |
+
+#### Column
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| field | string | ""  | Key of the data object. | |
+| title | string | ""  | Header name of the column. | |
+| visible | boolean |  true  | Show/Hide the column. | |
 
 ### Constructor
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:44]:
```suggestion

> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
```

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]

File: docs/document/docs/components/desktop/readonly-table.md [hunk 13]
```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
...
+| className | string | ""  | Component class name. | |
+| id | string | ""  | Component id name. | |
+| label | string | ""  | Text description for the component. | Label will not be displayed if unspecified or left empty. |
+| rowsPerPage | number | 5 | Number of table rows per page. | <ul><li>Round off to the nearest whole number. </li><ul> <li> `{rowsPerPage: 10}` Display up to 10 rows per page. </li><li> `{rowsPerPage: 20.5}` Display up to 21 rows per page. </li> <li> `{rowsPerPage: 20.4}` Display up to 20 rows per page. </li></ul><li>Throw an error if `rowsPerPage` is not a positive number in constructor and setter.</li><ul>|
+| pagination | boolean | true | Show/Hide the pagination. | <ul><li>`{pagination: false}` Pagination is hidden, all rows are displayed.</li><li>`{pagination: true}` Only the number of rows set in rowsPerPage are displayed.</li></ul> |
+| visible | boolean | true | Show/Hide the component. | |
+| columns | Array\<[Columns](#column)\> | []  | Column data of the component. | Throw an error if the `columns` is not an array. |
+| data | Array\<object\> | []  | Row data of the component. | Throw an error if the `data` is not an array. |
+
+#### Column
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/readonly-table.md:36]: How about writing inside "### Property" like this?
Column.field, Column.title, etc.
https://kintone-ui-component.netlify.app/docs/en/components/desktop/dropdown

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]

**Review by TomokoMiyake — APPROVED** [review]
I updated for review.
Please confirm with it.

**Pull request merged**